modules/chat/client/controllers/chat.client.controller.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
 // Create the 'chat' controller
-angular.module('chat').controller('ChatController', ['$scope', 'Socket',
-    function($scope, Socket) {
+angular.module('chat').controller('ChatController', ['$scope', '$location', 'Authentication', 'Socket',
```

Comment:
Yea... Once we decide to move completely away from $location, we can update this to use $state.go(). I just wasn't sure if this would be necessary to do atm; for the purposes of this fix.